### ISSUE TITLE
Voiceover issues with no auditory

### ISSUE THREAD

**Issue by asiliuk** [asiliuk/BsuirScheduleApp#91]
When VoiceOver read pair details that does not have auditory it says `Auditory:` 
It should not include this prefix if there's no auditory, or say `audiotry: unknown`

**Comment by asiliuk** [comment]
Can't reproduce on 2.8.0
